tests/infrastructure/golden_images/conftest.py

```
@@ -29,14 +28,10 @@ def updated_default_storage_class_scope_function(
-    """
-    Extract the version from file name, if no files found raise KeyError
-    file example: /tmp/pytest-6axFnW3vzouCkjWokhvbDi/osinfodb0/osinfo-db-20221121/os/fedoraproject.org/fedora-42.xml
-    """
-    osinfo_file_folder_path = os.path.join(downloaded_latest_libosinfo_db, "os/fedoraproject.org/")
-    list_of_fedora_os_files = list(sorted(Path(osinfo_file_folder_path).glob("*fedora-[0-9][0-9]*.xml")))
-    if not list_of_fedora_os_files:
-        raise FileNotFoundError("No fedora files were found in osinfo db")
-    latest_fedora_os_file = list_of_fedora_os_files[-1]
-    return re.findall(r"\d+", latest_fedora_os_file.name)[0]
+def latest_fedora_release_version():
+    response = requests.get(url="https://fedoraproject.org/releases.json", verify=False)
```

Ruff (0.14.3)

Probable use of `requests` call without timeout

(S113)